Detectors/Base/include/DetectorsBase/GeometryManager.h

```diff
@@ -73,6 +101,99 @@ class GeometryManager : public TObject
 
   ClassDefOverride(GeometryManager, 0); // Manager of geometry information for alignment
 };
+
+
+//_____________________________________________________________________________________
+template <typename T> 
```

Comment:
I am wondering why a template is required here? T is not really used anywhere in the implementation which is always double.

If you want to offer both `float` + `double` API, it might be better to be explicit about it and dispatch both of them to the same internal/private template kernel. At least nobody will then be able to pass `Point3D<bool>` to this function.